Script/char.dat

```diff
@@ -2111,8 +2376,23 @@ golem
   KnownCWeaponSkills == UNARMED;
   CWeaponSkillHits == 50;
   PanicLevel = 0;
-  HostileReplies == "Yes, master. Golem kill human. Golem then return."; /* No quotes! (golem engraves) */
-  FriendlyReplies == "Yes, master?";
+  HostileReplies = /* No quotes! (golem engraves) */
+  {
+    3,
+    "GOLEM KILL, GOLEM THEN RETURN",
```

Comment:
I was trying the avoid full stops for golems, that's why there is the comma instead.